### ISSUE TITLE
create "bare-metal" pytorch implementation of "academic paper classifier" tutorial

### ISSUE THREAD

**Issue by joelgrus** [allenai/allennlp#810]
this would help demonstrate our value prop (and is probably necessary for the tutorial we proposed)

**Comment by nelson-liu** [comment]
I can work on this now, if help is wanted here. I was planning on just mimicking the architecture in https://github.com/allenai/allennlp-as-a-library-example , but without some of the bells and whistles (e.g. re-implementing bucketing on bare metal does not sound fun).

**Comment by joelgrus** [comment]
we've been saving it just for you

**Comment by nelson-liu** [comment]
Ok, I have an initial version here: https://github.com/nelson-liu/pytorch-paper-classifier --- reviews are welcome! I ran the model, and I get a validation accuracy in the low-mid 80's after 4-8 epochs of training, which seems about in line with what's reported in [the AllenNLP tutorial](https://github.com/allenai/allennlp/blob/master/tutorials/getting_started/using_in_your_repo.md)

Not sure about the best way to review this --- you can either directly raise issues on that repo, or comment about things here.

After all's said and done, I'd be happy to transfer this to the allenai org if that's desired.

**Comment by schmmd** [comment]
@nelson-liu thanks!  This is fantastic!

I'll think about how to expose this from AllenNLP's documentation.

**Comment by schmmd** [comment]
My apologies for moving so slowly on this.  I'm not sure of the best way to make this accessible to our users.  It seems like it's best suited for a blog post rather than material on http://allennlp.org/tutorials.

@joelgrus did you have ideas for how to share this information?  Maybe it's time to start that blog...

**Comment by joelgrus** [comment]
I mostly wanted this to use in e.g. the EMNLP tutorial, but I think it would make for a good blog post and/or article.

**Comment by schmmd** [comment]
@joelgrus would this be helpful for the tutorial next week?
